contributors/devel/pull-requests.md

```diff
@@ -93,6 +93,9 @@ request that subsequently needs to be reopened. We want to limit the total numbe
 * Remove old PRs that would be difficult to rebase as the underlying code has changed over time
 * Encourage code velocity
 
+If the pull requests unfinished, prefix the title with "WIP-"; and tracking remaining todos
+in a checklist in the pull request description. k8s-bot must mark those pull request with
+`do-not-merge` to avoid unexpected mergers.
```

Comment:
don't mention the merge bot. That isn't something the community needs to deal with and I'm not sure I'm going to even write that feature.

Also some people use `WIP:` others use `WIP-` so I don't know if we need to be that prescriptive.  Maybe just say 'prefix the title with WIP'